### ISSUE TITLE
Add test to verify if EvaluateRequest works

### ISSUE THREAD

**Issue by afshin** [jupyterlab/debugger#419]
We should write a unit test to verify this works as a precursor to supporting watch expressions:
https://microsoft.github.io/debug-adapter-protocol/specification#Requests_Evaluate

cf. https://github.com/jupyterlab/debugger/issues/409

**Comment by jtpio** [comment]
The tests are defined in: https://github.com/jupyterlab/debugger/blob/master/tests/src/session.spec.ts

This would require adding a new `describe` block similar to the following one, but for the `evaluate` request:

https://github.com/jupyterlab/debugger/blob/e5cb406ff0e92c0a2f9aa5dff1f701583a74bb60/tests/src/session.spec.ts#L280-L295

**Comment by jtpio** [comment]
If anyone would like to implement this, here is the code that needs to be added to [`session.spec.ts`](https://github.com/jupyterlab/debugger/blob/master/tests/src/session.spec.ts):

```typescript
describe('#evaluate', () => {
  it('should evaluate the code sent to the kernel', async () => {
    const stackFramesReply = await debugSession.sendRequest('stackTrace', {
      threadId
    });
    const frameId = stackFramesReply.body.stackFrames[0].id;
    const reply = await debugSession.sendRequest('evaluate', {
      frameId,
      context: 'repl',
      expression: 'k = 123',
      format: {}
    });
    expect(reply.success).toBe(true);

    const variables = await getVariables();
    const k = find(variables, variable => variable.name === 'k');
    expect(k).toBeDefined();
    expect(k.type).toEqual('int');
    expect(k.value).toEqual('123');
  });
});
```

**Comment by jtpio** [comment]
Fixed by #472.
